src/set-cookies.js

```diff
@@ -0,0 +1,14 @@
+import {registerFsReady} from "./utils/fs";
+import {setCookiesAsUserVars} from "./utils/cookie";
+
+/**
+ *
+ * set cookie names as user variables depending on window._fs_cookies_setUserVar
+ * Please note you must set the window._fs_cookies_setUserVar variable for this to have effect
+ * Variable should be an array of cookie names to turn into userVars
+ */
+registerFsReady( () => {
+  if( window._fs_cookies_setUserVar ){
+    setCookiesAsUserVars( window._fs_cookies_setUserVar );
```

Comment:
Can we also expand this to also include page vars?  This way you could define the cookies of interest and send them to the desired FS API.

Reply:
Absolutely.  Wasn't sure if I should add that upfront or wait for a use case.

Reply:
Code implemented below